manifests/n/NumeRe/NumeRe/1.1.3.1/NumeRe.NumeRe.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3.1

```

Comment:
```suggestion
PackageVersion: 1.1.3.1-hf1
```

Reply:
Hello @OfficialEsco!

Thanks for mentioning that. Will be fixed in installer in a few minutes. I'll commit an updated manifest with the correspondung SHA